packages/opencensus-instrumentation-http2/test/test-http2.ts

```diff
@@ -100,12 +99,14 @@ describe('Http2Plugin', () => {
   before(() => {
     tracer.registerSpanEventListener(rootSpanVerifier);
 
-    plugin.enable(http2, tracer, VERSION, {}, null);
+    plugin.enable(http2, tracer, VERSION, {}, '');
     server = http2.createServer();
     server.on('stream', (stream, requestHeaders) => {
-      const statusCode = requestHeaders[':path'].length > 1 ?
-          +requestHeaders[':path'].slice(1) :
-          200;
+      const path = requestHeaders[':path'];
+      let statusCode = 200;
+      if (path) {
+        statusCode = path.length > 1 ? +path.slice(1) : 200;
```

Comment:
Nit: what would you think about using `Number(path.slice(1))` to make the type conversion a bit more explicit?

And do we need to worry about malformed numbers here (that would give `NaN` to the `Number` constructor call)? If so, I'd say we can do in a follow up PR, but figured I'd mention as I'm thinking of it here.

Reply:
Good call, added ```isNaN``` check. PTAL